src/DataStreams/RemoteQueryExecutor.cpp

```diff
@@ -233,7 +266,29 @@ std::variant<Block, int> RemoteQueryExecutor::read(std::unique_ptr<ReadContext>
         else
         {
             if (auto data = processPacket(std::move(read_context->packet)))
+            {
+                if (got_duplicated_part_uuids)
```

Comment:
maybe I should have moved it to an inline function to avoid duplication above https://github.com/ClickHouse/ClickHouse/pull/17348/files#diff-2fce06be43da8c510783b4c0ca49218697250dc38c15da259e417c79ac17507eR210

Reply:
Resolved.